charts/graphiti-mcp/.markdownlint.yaml

```
@@ -0,0 +1,272 @@
+# MD041/first-line-heading/first-line-h1 : First line in a file should be a top-level heading : https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md041.md
+MD041:
+  # Heading level
+  level: 1
+  # RegExp for matching title in front matter
+  front_matter_title: "^\\s*title\\s*[:=]"
+
+# MD042/no-empty-links : No empty links : https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md042.md
+MD042: true
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (3 > 2)

(empty-lines)